Library/Homebrew/cmd/uses.rb

```diff
@@ -125,4 +131,44 @@ def select_used_dependents(dependents, used_formulae, recursive, includes, ignor
       end
     end
   end
+
+  def puts_dependents_tree(use_runtime_dependents, used_formulae, args:)
+    used_formulae.sort_by(&:full_name).each do |f|
+      puts f.full_name
+    end
+    @dependents_stack = []
+    recursive_dependents_tree(use_runtime_dependents, used_formulae, "", args: args)
+  end
+
+  def recursive_dependents_tree(use_runtime_dependents, formulae, prefix, args:)
```

Comment:
Could any of this logic be shared with `deps`?

Reply:
I think so, I just need to figure out how to handle differentiating between finding the dependencies of each given formula independently (`deps`) and finding the intersection of dependents of all given formulae (`uses`)

Reply:
@SeekingMeaning I think I'd be more game for this functionality if we find a way of sharing this code.

Reply:
Definitely, I just need to figure out a more abstract view of both commands. Closing this for now.